### PR TITLE
[60] [UI] Smiley rating screen

### PR DESCRIPTION
**Pull request by AVI5HEK**
https://github.com/nimblehq/ic-flutter-avishek/issues/60

## What happened 👀

- Show the smiley emoji - user can select only one at a time.
- Show the emoji type based on the question's `display_type` (smiley).

## Insight 📝

N/A.

### Android

https://github.com/nimblehq/ic-flutter-avishek/assets/8093908/417c8078-f95a-47b9-9a17-37a4bb72f186

### iOS

https://github.com/nimblehq/ic-flutter-avishek/assets/8093908/0b94ffef-cf25-42d0-848f-1c261de147dc

### CODE REVIEW

File: lib/ui/surveydetail/survey_question_content.dart [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 import 'package:flutter_rating_bar/flutter_rating_bar.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_survey/ui/surveydetail/single_selectable_rating_bar.dart';
+import 'package:flutter_survey/utils/extension/iterable_ext.dart';
 
 import '../../model/question.dart';
 
 const starEmoji = "⭐️️";
 const heartEmoji = "❤️️";
+const smileys = {0: "😡", 1: "😕", 2: "😐", 3: "🙂", 4: "😄"};
```
Comment (markgravity) [inline lib/ui/surveydetail/survey_question_content.dart:11]: What do you think about marking as private for these constants? Or we should move it to a constant file if we want to public them 🙏

Comment (AVI5HEK) [reply lib/ui/surveydetail/survey_question_content.dart]: Done. https://github.com/nimblehq/ic-flutter-avishek/pull/61/commits/a6d813955431c1c0e0cb5abfc7ef39a7903a462b

**Review by markgravity — APPROVED** [review]

**Comment by Wadeewee** [comment]
Conflict 🦀

**Comment by AVI5HEK** [comment]
> Conflict 🦀

@Wadeewee resolved.

**Review by Wadeewee — APPROVED** [review]

**Pull request merged**